server/src/main/java/org/apache/druid/curator/CuratorConfig.java

```diff
@@ -52,6 +52,10 @@
   @JsonProperty("authScheme")
   private String authScheme = "digest";
 
+  @JsonProperty("quitOnConnectFail")
```

Comment:
IMO "quit" is a little vague. Maybe "terminateDruidProcess".

Reply:
Please add documentation for this property.

Reply:
Fixed.

Added documentation.